Core/ContentBlockerRulesBuilder.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  ContentBlockerRulesBuilder.swift
+//  DuckDuckGo
+//
+//  Copyright © 2019 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public struct ContentBlockerRulesBuilder {
```

Comment:
Both this and `ContentBlockerRule.swift` seem to have plenty of non-trivial logic. Maybe some unit/simple integration tests could be added for important flows?